package.json

```diff
@@ -40,7 +40,9 @@
   "devDependencies": {
     "@commitlint/cli": "19.6.0",
     "@commitlint/config-conventional": "19.6.0",
+    "@endo/compartment-mapper": "1.4.0",
```

Comment:
Like other packages, we are pulling this in only to set the [override](https://github.com/LavaMoat/LavaMoat/pull/772/files/eb70b9cf1070a47baa2ae5209fac1c858e34a08b#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R73) below. This ensures `@endo/compartment-mapper` is the same version across all workspaces.